.github/workflows/test_inc.yml

```
@@ -35,7 +35,8 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install torch==${{ matrix.torch-version }} torchaudio torchvision --index-url https://download.pytorch.org/whl/cpu
-          pip install .[neural-compressor,diffusers,peft,tests] transformers[testing] intel-extension-for-pytorch==${{ matrix.torch-version }}
+          pip install .[neural-compressor,peft,tests] transformers[testing] intel-extension-for-pytorch==${{ matrix.torch-version }}
```

no peft extra so removing, also removing transformers as the transformers release is limited https://github.com/huggingface/optimum-intel/blob/main/setup.py#L70

```suggestion
          pip install .[neural-compressor,tests] intel-extension-for-pytorch==${{ matrix.torch-version }}
```